apps/table.html

```diff
@@ -142,13 +134,25 @@ <h1 class="h1">caMicroscope</h1>
 			<div style="align-content: center;">
 				<div class="btn-group" role="group">
 				<button type="button" style="border-color: white;" class="btn btn-secondary bg-white text-dark" title="Slides Table"><i class="fas fa-list-alt"></i>  Slides</button>
-				<a href="./Info.html">	<button style="border-color: white; border-radius: 0 5px 5px 0;" type="button" class="btn btn-secondary bg-info text-light" title="Information Dashboard"><i class="fas fa-info-circle"></i>  Info</button> </a></div>
+				<a href="./info.html">	<button style="border-color: white; border-radius: 0 5px 5px 0;" type="button" class="btn btn-secondary bg-info text-light" title="Information Dashboard"><i class="fas fa-info-circle"></i>  Info</button> </a></div>
```

Comment:
```suggestion
				<a href="./Info.html">	<button style="border-color: white; border-radius: 0 5px 5px 0;" type="button" class="btn btn-secondary bg-info text-light" title="Information Dashboard"><i class="fas fa-info-circle"></i>  Info</button> </a></div>
```

Reply:
Really good point. I actually changed all the links to info.html after changing the name to 'info.html' in order to maintain the naming convention. But due to some reason (maybe I forgot) it didn't reflect in the PR. Shall I rename it and commit again, or change all the links to Info.html back? What do you prefer?